routes/photos.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const router = express.Router();
+const handleError = require('../helpers/handleError');
+const processInput = require('../helpers/processInput');
+const photosQueries = require('../queries/photosQueries')
+
+
+router.get('/', async (req, res, next) => {
+    try {
+        let allPhotos = await photosQueries.getAllPhotos()
+        res.json({
+            status: "success",
+            message: "all photos retrieved",
+            payload: allPhotos
+        });
+    } catch (err) {
+        handleError(err, req, res, next);
+    }
+});
+
+
+router.get('/:id', async (req, res, next) => {
+    try {
+        const id = processInput(req.params.id, "idNum", "photo id");
+        let photo = await photosQueries.getPhotosByReclaimId(id)
+        res.json({
+            status: "success",
+            message: "Retrieved photo from a specific reclaim id",
+            payload: photo,
+            err: false
```

Comment:
@KathyPuma I saw either you or Aransa add this err: false to the response.json another time earlier. What is it for?

Reply:
I don't think it affects anything good or bad. Just curious